harness-delegate-ng/templates/service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: delegate-service
+  name: {{ include "harness-delegate-ng.fullname" . }}
```

Comment:
I don't have CI, I'm interested only in the deploy and this Service is created in any case

If that's CI is the case there should have some condition on it.

Reply:
That sounds right, it should be created conditionally since it's not used if you don't have CI enabled. Can you propose such change instead?
CC @AnupamIO he is probably best to follow up on helm chart changes.

Reply:
BTW CI team is currently in the process of retiring this service as a requirement altogether. AFAIK it's currently used as a fallback mechanism only and not by default. 
This means we still need it for some time, but should definitely be configurable and possibly even off by default.